UnleashClient/__init__.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         self.unleash_app_name = app_name
         self.unleash_environment = environment
         self.unleash_instance_id = instance_id
+        self.connection_id = str(uuid.uuid4())
```

Comment:
we had connection id before. is this code moved?

Reply:
Yes, it was initialized only in headers before, now we can use it also in body if needed.

https://github.com/Unleash/unleash-client-python/pull/346/files#diff-d8684313c6cfe7bbd96df3385f5b28c7f337cd5159fc08f79b518df75cc13475L218